LOAN/LoanContracts/LoanHMDA.rdf

```diff
@@ -151,47 +201,33 @@ Copyright (c) 2015-2018 Object Management Group, Inc.</sm:copyright>
 		<fibo-fnd-utl-av:adaptedFrom>2015 Revised HMDA regulation</fibo-fnd-utl-av:adaptedFrom>
 	</owl:Class>
 	
-	<owl:NamedIndividual rdf:about="&fibo-loan-ln-hmda;_HMDA_Disposition_ApplicationApprovedButNotAccepted">
-		<rdf:type rdf:resource="&fibo-loan-ln-hmda;HMDA_Disposition"/>
-	</owl:NamedIndividual>
-	
-	<owl:NamedIndividual rdf:about="&fibo-loan-ln-hmda;_HMDA_Disposition_ApplicationDenied">
-		<rdf:type rdf:resource="&fibo-loan-ln-hmda;HMDA_Disposition"/>
-	</owl:NamedIndividual>
-	
-	<owl:NamedIndividual rdf:about="&fibo-loan-ln-hmda;_HMDA_Disposition_ApplicationWithdrawn">
-		<rdf:type rdf:resource="&fibo-loan-ln-hmda;HMDA_Disposition"/>
-	</owl:NamedIndividual>
-	
-	<owl:NamedIndividual rdf:about="&fibo-loan-ln-hmda;_HMDA_Disposition_FileClosedForIncompleteness">
-		<rdf:type rdf:resource="&fibo-loan-ln-hmda;HMDA_Disposition"/>
-	</owl:NamedIndividual>
-	
-	<owl:NamedIndividual rdf:about="&fibo-loan-ln-hmda;_HMDA_Disposition_LoanOriginated">
-		<rdf:type rdf:resource="&fibo-loan-ln-hmda;HMDA_Disposition"/>
-	</owl:NamedIndividual>
-	
-	<owl:NamedIndividual rdf:about="&fibo-loan-ln-hmda;_HMDA_Disposition_LoanPurchasedByYourInstitution">
-		<rdf:type rdf:resource="&fibo-loan-ln-hmda;HMDA_Disposition"/>
-	</owl:NamedIndividual>
-	
-	<owl:NamedIndividual rdf:about="&fibo-loan-ln-hmda;_HMDA_Disposition_PreapprovalRequestApprovedButNotAccepted">
-		<rdf:type rdf:resource="&fibo-loan-ln-hmda;HMDA_Disposition"/>
-	</owl:NamedIndividual>
-	
-	<owl:NamedIndividual rdf:about="&fibo-loan-ln-hmda;_HMDA_Disposition_PreapprovalRequestDenied">
-		<rdf:type rdf:resource="&fibo-loan-ln-hmda;HMDA_Disposition"/>
-	</owl:NamedIndividual>
-	
-	<owl:NamedIndividual rdf:about="&fibo-loan-ln-hmda;_HowSubmitted_directly">
+	<owl:NamedIndividual rdf:about="&fibo-loan-ln-hmda;HowSubmitted-directly">
```

Comment:
I'm assuming that this means that the submission was to Fannie Mae or some related organization rather than through an intermediary, but I would need to ask Lynn Calahan or Jennifer Caswell.